ROMFS/px4fmu_common/init.d/airframes/1100_rc_quad_x_sih.hil

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# @name SIH Quadcopter X
+#
+# @type Simulation
```

Comment:
FYI, you can see how this will render here: https://hamishwillee.gitbooks.io/havdevguide/content/v/test_sih/en/airframes/airframe_reference.html#simulation-copter

Reply:
great!